.vscode/launch.json

```diff
@@ -2,35 +2,34 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-
     "version": "0.2.0",
     "configurations": [
-
         {
             "name": "Kitsune",
             "type": "python",
-            "request": "launch",
+            "request": "",
```

Comment:
This is an invalid value, vscode throws an error: "Attribute 'request' is missing from the chosen debug configuration".